tuna/connection.py

```diff
@@ -106,22 +119,25 @@ def get_bin_str(cmd):
 
     return bin_str
 
-  def test_cmd_str(self, cmd):
+  def test_cmd_str(self, cmd: str) -> bool:
     """Function to look for installed binary"""
+    split_cmd: Any
```

Comment:
str initially and then list
if we're removing ambiguous typing, change this to two variables.
The first is needed to make a mutable copy of cmd.
The second is for a component list.